electron/menu.ts

```diff
@@ -47,12 +47,19 @@ const appMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions =>
   };
 };
 
-const fileMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions => {
+const fileMenu = (store: any): MenuItemConstructorOptions => {
   const configState: AppConfig = store.getState().config;
   const mainState: AppState = store.getState().main;
   return {
     label: 'File',
     submenu: [
+      {
+        label: 'New Window',
```

Comment:
"New Monokle Window" ?

Reply:
Do we need a corresponding "Close Window" in the Window menu?

Reply:
Hmm I can add it. No big deal